packages/api/src/memberContext.ts

```diff
@@ -120,12 +133,18 @@ export class MemberContext implements MemberContext {
 
   mailContext: MailContext
 
+  invoiceReminderFrequencyInDays: number
+  invoiceReminderMaxTries: number
+
   constructor(props: MemberContextProps) {
     this.dbAdapter = props.dbAdapter
     this.loaders = props.loaders
     this.paymentProviders = props.paymentProviders
 
     this.mailContext = props.mailContext
+
+    this.invoiceReminderFrequencyInDays = parseInt(process.env.INVOICE_REMINDER_FREQ ?? '') ?? 3
```

Comment:
Should we add these env keys to .env.example?

Reply:
No. We only add envs that are needed to run we.publish to `.env.example`